UserAdminBundle/Controller/Admin/UserController.php

```diff
@@ -79,8 +82,10 @@ public function formSelfAction(Request $request)
         if (!$this->get('security.authorization_checker')->isGranted('IS_AUTHENTICATED_FULLY')) {
             throw $this->createAccessDeniedException();
         }
-
+        $userId = $this->get('security.token_storage')->getToken()->getUser()->getId();
```

Comment:
i don't understand this part. And you can use $this->getUser() inside a controller instead $user = $this->get('security.token_storage')->getToken()->getUser();